clients/avro/src/main/java/io/confluent/examples/clients/basicavro/ProducerExample.java

```diff
@@ -35,13 +36,14 @@ public static void main(final String[] args) {
                 Thread.sleep(1000L);
             }
 
+            System.out.printf("Successfully produced 10 messages to a topic called %s%n", TOPIC);
```

Comment:
There should be a flush above this line.

Reply:
Ack and done